site/content/en/docs/Guides/Client SDKs/rest.md

```diff
@@ -238,6 +238,105 @@ Apply an Annotation with the prefix "agones.dev/sdk-" to the backing `GameServer
 curl -d '{"key": "foo", "value": "bar"}' -H "Content-Type: application/json" -X PUT http://localhost:${AGONES_SDK_HTTP_PORT}/metadata/annotation
 ```
 
+### Counters and Lists
+
+{{< alpha title="Counters and Lists" gate="CountsAndLists" >}}
+
+#### Counters
+
+In all the  Counter examples, we retrieve the counter under the key `rooms` as if it was previously defined in [`GameServer.Spec.counters[room]`]({{< ref "/docs/Reference/agones_crd_api_reference.html#agones.dev/v1.GameServerSpec" >}}).
+
+For your own Counter REST requests, replace the value `rooms` with your own key in the path.
+
+##### Alpha: GetCounter
+This function retrieves a specified counter by its key, `rooms`, and returns its information.
+
+###### Example
+
+```bash
+curl -H "Content-Type: application/json" -X GET http://localhost:${AGONES_SDK_HTTP_PORT}/v1alpha1/counters/rooms
+```
+
+Response:
+```json
+{"name":"rooms", "count":"1", "capacity":"10"}
+```
+
+##### Alpha: UpdateCounter
+This function updates the properties of the counter with the key `rooms`, such as its count and capacity, and returns the updated counter details.
+
+###### Example
+
+```bash
+curl -d '{"count": "5", "capacity": "11"}' -H "Content-Type: application/json" -X PATCH http://localhost:${AGONES_SDK_HTTP_PORT}/v1alpha1/counters/rooms
+```
+
+Response:
+```json
+{"name":"rooms", "count":"5", "capacity":"11"}
+```
+
+#### Lists
+
+In all the List examples, we retrieve the list under the key `players` as if it was previously defined in [`GameServer.Spec.lists[players]`]({{< ref "/docs/Reference/agones_crd_api_reference.html#agones.dev/v1.GameServerSpec" >}}).
+
+For your own List REST based requests, replace the value `players` with your own key in the path.
+
+##### Alpha: GetList
+This function retrieves a specific list identified by its name, returns the list's information.
+
+###### Example
+```bash
+curl -H "Content-Type: application/json" -X GET http://localhost:${AGONES_SDK_HTTP_PORT}/v1alpha1/lists/players
+```
+
+Response:
+```json
+{"name":"players", "capacity":"100", "values":["player0", "player1", "player2"]}     
+```
+
+##### Alpha: UpdateList
+This function updates the specified list's properties, such as its capacity and values, returns the updated list details. This will overwrite all existing List.Values with the update list request values. Use addValue or removeValue for modifying the List.Values field.
```

Comment:
Can we please do `players` in this List section like we did for `rooms` above, and then I think this PR is good to go.

```suggestion
This function updates the list's properties with the key `players`, such as its capacity and values, returns the updated list details. This will overwrite all existing List.Values with the update list request values. Use addValue or removeValue for modifying the List.Values field.
```